README.md

```diff
@@ -261,10 +262,23 @@ app.ready(function (err, context, done) {
 
 Returns the instance on which `ready` is called, to support a chainable API.
 
+-------------------------------------------------------
+<a name="wait"></a>
+
+### app.wait()
+If you are using async await, you can avoid the use of `after` and `ready`, and just use the `wait` api.
```

Comment:
If you are using async await, you can avoid the use of `after` and `ready`, and just use the `wait` api which returns a promise.